core/plugins/org.polarsys.capella.core.projection.exchanges/src/org/polarsys/capella/core/projection/exchanges/PhysicalLinksCreator.java

```diff
@@ -160,42 +165,70 @@ protected void createPhysicalLinksFromCExchanges(Component sourceContainer, Comp
     }
   }
   
+  private boolean isContainedByALogicalComponent(Component component) {

```

Comment:
Can you add some comments to explain what it does? 
return true if the given component/actor is owned by a not actor component right ?

Reply:
yes :)